content/projects/ios-mobile/unit-testing/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+available_flavours:
+- swift
+content_type: project
+prerequisites:
+  hard: []
+  soft: []
+ready: true
+submission_type: repo
+title: TDD and Unit Testing
+---
+
+### Useful Content to Snack on: 
+
+[How to approach TDD and write unit tests](https://www.appsdeveloperblog.com/swift-tdd-how-many-unit-tests-to-write/)
```

Comment:
Would it not be useful to put tis in a seperate TOPIC?

Also, it's often good to give peopel specific instructions. Like: "Please read and understand the following content: "

Sometimes recruits jut ignore links unless you tell them what to do with them

Reply:
On it